fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/SelectMaterializedIndexWithAggregate.java

```diff
@@ -1087,7 +1084,7 @@ private static class ExprRewriteMap {
         /**
          * Replace map for expressions in project.
          */
-        public final Map<Expression, Expression> projectExprMap;
+        public final Map<Expression, List<Expression>> projectExprMap;
```

Comment:
please add a sample to explain the problem of multiple aggs with same slot, and how to solve it in this pr